modules/ROOT/pages/set-anypoint-platform-cloud.adoc

```diff
@@ -0,0 +1,20 @@
+= Configure Anypoint Platform Cloud in Studio
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can deploy your Mule applications from Anypoint Studio to the Anypoint Platform cloud. 
+
+From Studio, you can:
+
+* xref:pce3-configuration.adoc[Configure Anypoint Platform PCE 3]
```

Comment:
Are all these meant to be sentences or topic tiles?

Reply:
I meant to add links to the cloud options Studio offers, so they are topic titles. 
Did I miss something?

Reply:
Just that one of them was sentence case.

Reply:
Oh! And they are all titles now! I got it! Thanks!